pallets/randomness/src/benchmarks.rs

```diff
@@ -0,0 +1,158 @@
+// Copyright 2019-2022 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+//! Benchmarking
+use crate::{BalanceOf, Config, Pallet, RandomnessResults, Request, RequestType};
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite, Zero};
+use frame_support::traits::{Currency, Get};
+use pallet_evm::AddressMapping;
+use sp_core::{H160, H256};
+use sp_runtime::traits::One;
+
+/// Create a funded user from the input
+fn fund_user<T: Config>(user: H160, fee: BalanceOf<T>) {
+	let total_minted = fee + <<T as Config>::Deposit as Get<BalanceOf<T>>>::get();
+	T::Currency::make_free_balance_be(&T::AddressMapping::into_account_id(user), total_minted);
+	T::Currency::issue(total_minted);
+}
+
+benchmarks! {
```

Comment:
I've had trouble running benchmarks on the benchmarking server recently. It gets disconnected during compilation and doesn't finish compiling.

@notlesh @mario-purestake Can I use the bench bot for this?

Need to run benchmarks to compute weights, then need to convert those weights into gas to charge in the precompile.